src/ocaml/typing/oprint.ml

```diff
@@ -541,6 +541,7 @@ and print_simple_out_module_type ppf =
   | Omty_alias id -> fprintf ppf "(module %a)" print_ident id
   | Omty_functor _ as non_simple ->
      fprintf ppf "(%a)" print_out_module_type non_simple
+  | Omty_hole -> fprintf ppf "'S"
```

Comment:
Is that supposed to represent a signature variable?

Reply:
Yes, the intention is to mean "Any signature". Do you have a better solution ? Maybe an explicit name like "Any_sig" ?
Note that `sig end` carries a different meaning. Maybe `sig _ end` ?